lib/graphql/execution/interpreter/resolve.rb

```diff
@@ -25,8 +25,10 @@ def self.resolve_all(results)
         # or return {Hash}/{Array} if the query should be continued.
         #
         # @param results [Array]
-        # @return [Array] Same size, filled with finished values
+        # @return [Array] The next round of lazies to resolve
         def self.resolve(results)
+          # First, kick off any loaders that will resolve in background threads
+          Dataloader.current && Dataloader.current.process_async_loader_queue
```

Comment:
This isn't exactly subtle, but after a lot of attempts, I couldn't find a better way to work in a "kick off" step into the existing execution flow. This is very similar to the [original suggestion](https://github.com/rmosolgo/graphql-ruby/pull/1981/files#r238053370), but at the loader level instead of the promise level.